### PR TITLE
Fix client gem publishing for the release CI

### PR DESCRIPTION
**Pull request by quba42**
[noissue]

### CODE REVIEW

**Review by mdellweg — APPROVED** [review]

File: .ci/scripts/publish_plugin_pypi.sh [hunk 1]
```diff
@@ -12,6 +12,14 @@ cd "$(dirname "$(realpath -e "$0")")"/../..
 
 set -euv
 
+export VERSION=$(http pulp/pulp/api/v3/status/ | jq --arg plugin deb -r '.versions[] | select(.component == $plugin) | .version')
+export response=$(curl --write-out %{http_code} --silent --output /dev/null https://pypi.org/project/pulp-deb-client/$VERSION/)
```
Comment (mdellweg) [inline .ci/scripts/publish_plugin_pypi.sh:16]: This looks wrong.

File: .ci/scripts/publish_plugin_pypi.sh [hunk 2]
```diff
@@ -12,6 +12,14 @@ cd "$(dirname "$(realpath -e "$0")")"/../..
 
 set -euv
 
+export VERSION=$(http pulp/pulp/api/v3/status/ | jq --arg plugin deb -r '.versions[] | select(.component == $plugin) | .version')
```
Comment (mdellweg) [inline .ci/scripts/publish_plugin_pypi.sh:15]:
```suggestion
export VERSION=$(http pulp/pulp/api/v3/status/ | jq --arg plugin pulp_deb -r '.versions[] | select(.component == $plugin) | .version')
```
Assuming this is using the old status api...

File: .ci/scripts/publish_plugin_pypi.sh (hunk 1 above)
Comment (quba42) [reply .ci/scripts/publish_plugin_pypi.sh]: I dropped the `-client` like you did here: https://github.com/pulp/plugin_template/pull/371/files

File: .ci/scripts/publish_client_pypi.sh [hunk 3]
```diff
@@ -32,6 +32,7 @@ export response=$(curl --write-out %{http_code} --silent --output /dev/null http
 
 if [ "$response" == "200" ];
 then
+  echo "pulp_deb $VERSION has already been released. Skipping."
```
Comment (mdellweg) [inline .ci/scripts/publish_client_pypi.sh:35]: The version check here is broken too.

**Comment by quba42** [comment]
Now it is simply the latest plugin template (with your fix) applied to this branch.

File: .ci/scripts/publish_plugin_pypi.sh [hunk 4]
```diff
@@ -12,6 +12,15 @@ cd "$(dirname "$(realpath -e "$0")")"/../..
...
```
Comment (mdellweg) [inline .ci/scripts/publish_plugin_pypi.sh:15]: Hmm this line ought to be deleted...

**Review by mdellweg — APPROVED** [review]

File: .ci/scripts/publish_plugin_pypi.sh (hunk 4 above)
Comment (quba42) [reply .ci/scripts/publish_plugin_pypi.sh]: Done.

**Pull request merged**